_includes/members/IMS.html

```diff
@@ -11,13 +11,13 @@
     The foundation scope of the Italian Mars Society is to promote research projects devoted to Mars Exploration
     and the involvement of SMEs and large enterprises in the new economy related to space exploration.
 
-    IMS is currently spearheading the European
-    <a href="{{ site.ims_eras_url }}">MaRs Analog Station for Advanced Technologies Integration</a> (ERAS) project.
+    IMS is currently spearheading the
+    <a href="{{ site.ims_eras_url }}">MARS CITY</a> project.
```

Comment:
@mtambos - could you check this change is correct? Also, I removed the mailing list as it seems not accessible anymore.